app/views/core_induction_programmes/modules/show.html.erb

```diff
@@ -28,5 +28,20 @@
         </ul>
       </li>
     </ol>
+    <nav class="step-by-step-navigation" role="navigation" aria-label="Lesson Part Navigation">
+      <ul class="step-by-step-navigation__list">
+        <% if @course_module.previous_module %>
+          <%= render NavigationPreviousComponent.new(
+            url: year_module_path(id: @course_module.previous_module.id),
```

Comment:
Does `year_module_path(course_module.previous_module)` work?

Reply:
No, it just reloads the same page